package.json

```diff
@@ -72,12 +72,13 @@
     "languages": [
       {
         "id": "quarkus-properties",
+        "filenamePatterns": [
+          "application.properties",
+          "application-?*.properties"
```

Comment:
This ensures the language id is set correctly. However, I noticed that the extension doesn't start until the file is opened, while with just application.properties it does.

Is it in the scope of this change to add it to activationEvents or does that not support globs in this way.

Reply:
It's definitely in scope. I'll take look into it

Reply:
Okay, should be fixed